GOVERNANCE.md

```diff
@@ -1,45 +1,124 @@
 
 ## Code of Conduct
 
-The TiKV community follows the TiKV Code of Conduct. Here are some excerpts:
+The TiKV community follows the [TiKV Code of Conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md). Here are some excerpts:
 
-> In the interest of fostering an open and welcoming evironment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
 
-## Community Membership
+## Community Roles and Responsibilities
 
-Refer to [Community Membership](community-membership.md).
+Refer to [Community Membership](/community-membership.md).
 
-## Community Structure
+## Community Groups
 
 ![community organization](/media/governance/community_organization.png)
 
-The community structure comprises two parts. The first part is the community organization, and the second part is the role division within the community group and the promotion path of each community role.
+The project is comprised of and operated by the following subgroups:
+
+- Project Management Committee (PMC)
+- Organizer Committee
+- Developer Group
+    - Maintainers
+    - Special Interest Groups, SIGs (Tech Leads)
+    - Working Groups, WGs (Organizer)
+
+### Project Management Committee
 
-PMC: The Project Management Committee; functions as the core management team that oversees the TiDB community. The PMC decides the roadmap of the project and makes major decisions related to the community.
+The PMC functions as the core management team that oversees the TiKV community. The PMC has additional responsibilities over and above those of Maintainers. These responsibilities ensure the smooth running of the project.
 
-Developer Group:
+Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new Maintainers or Committers, and makes all major decisions for the future with respect to TiKV, such as project-level governance policies,management of sub-structures, security processes and so on. It also makes decisions when community consensus cannot be reached. In addition, the PMC has access to the project’s private mailing list and its archives.
```

Comment:
```suggestion
Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new Maintainers or Committers, and makes all major decisions for the future with respect to TiKV, such as project-level governance policies, management of sub-structures, security processes and so on. It also makes decisions when community consensus cannot be reached. In addition, the PMC has access to the project’s private mailing list and its archives.
```

Reply:
We should note that the PMC executed the ban/exile folks who violate the Code of Conduct, or in special cases (such as sabotage) as found by a supermajority of the PMC.